### PR TITLE
U N I C O D E

### PR DESCRIPTION
**Pull request by laurie-dgit**
unnnnniiiiiccccoooooddddeeeeee

### CODE REVIEW

**Comment by mhanrahan** [comment]
Looks Good.

File: 🜚 .xml [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+            <ID>BOD-SPC000000000061</ID>
+            <Name>BoD P2P New Order</Name>
+            <Version>1.0</Version>
+            <Process>New-Order-v1-0</Process>
+        </SpecifiedBy>
+        <ContractSummary>
+            <ContractStartDate>2018-11-27T13:00:00Z</ContractStartDate>
+            <ContractEndDate>2018-12-07T13:00:00Z</ContractEndDate>
+        </ContractSummary>
+        <InteractionStatus>
```
Comment (mhanrahan) [inline 🜚 .xml:18]: InteractionStatus? Really? In this economy?

File: 🜚 .xml [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+                        </OfferingDescribes>
+                        <Class>
+                            <ID>L1 Circuit</ID>
+                        </Class>
+                        <Status>Draft</Status>
+                        <IsManaged>true</IsManaged>
+                        <LastModificationDate>2018-11-27T03:21:12.783666Z</LastModificationDate>
+                        <OperationalStatus>
+                            <ID>Feasibility Failed</ID>
+                            <Name>Feasibility Failed</Name>
```
Comment (mhanrahan) [inline 🜚 .xml:400]: Fail fail faiil

File: 🜚 .xml [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+                        <OfferingDescribes>
+                            <ID>BOD-PRO000000000075</ID>
+                        </OfferingDescribes>
+                        <Class>
+                            <ID>L1 Circuit</ID>
+                        </Class>
+                        <Status>Draft</Status>
+                        <IsManaged>true</IsManaged>
+                        <LastModificationDate>2018-11-27T03:21:12.783666Z</LastModificationDate>
+                        <OperationalStatus>
```
Comment (mhanrahan) [inline 🜚 .xml:398]:
```suggestion
                        <BOOperationalStatus>
```

**Review by mhanrahan — CHANGES_REQUESTED** [review]

File: 🜚 .xml (hunk 1 above)
Comment (laurie-dgit) [reply 🜚 .xml:18]: blame marketing

**Review by mhanrahan — DISMISSED** [review]
I approve now!

**Comment by laurie-dgit** [comment]
Now with added × 2 🜚

**Review by mhanrahan — APPROVED** [review]
ANOTHER ONE.  FINE.

**Pull request merged**